locksmith/src/controllers/hookController.ts

```diff
@@ -0,0 +1,131 @@
+// eslint-disable-next-line import/no-unresolved
+import { Response } from 'express-serve-static-core'
+import { networks } from '@unlock-protocol/networks'
+import * as z from 'zod'
```

Comment:
Very interesting! I did not know about zod!

Reply:
Yeah, it's a good way to declare schema which you can enforce/parse for as well as infer the type for it.